front/src/applications/stdcmV2/components/StdcmResults/StdcmSimulationNavigator.tsx

```diff
@@ -0,0 +1,105 @@
+import { formatDateToString } from 'utils/date';
+
+export const SIMULATION_ITEM_CLASSNAME = 'simulation-item';
+const ITEM_TO_SHOW_COUNT_ON_SCROLL = 3;
+
+type StdcmSimulationNavigatorProps = {
+  simulationsList: StdcmSimulation[];
+  retainedSimulationIndex: number;
+  selectedSimulationIndex: number;
+  onSelectSimulation: (simulationIndex: number) => void;
+};
+
+const StdcmSimulationNavigator = ({
+  simulationsList,
+  retainedSimulationIndex,
+  selectedSimulationIndex,
+  onSelectSimulation,
+}: StdcmSimulationNavigatorProps) => {
+  const { t } = useTranslation('stdcm', { keyPrefix: 'simulation.results' });
+
+  const { scrollableRef, showLeftBtn, showRightBtn, scrollLeft, scrollRight } = useHorizontalScroll(
+    SIMULATION_ITEM_CLASSNAME,
+    ITEM_TO_SHOW_COUNT_ON_SCROLL
+  );
+
+  return (
+    <div className="simulation-navigator">
+      <div className="simulation-list-wrapper">
+        {showLeftBtn && (
+          <div
+            className="scroll-btn left"
+            role="button"
+            tabIndex={0}
+            aria-label="Scroll left"
+            onClick={scrollLeft}
+          >
+            <span>
+              <ChevronLeft />
+            </span>
+          </div>
+        )}
+        <div className="simulation-list" ref={scrollableRef}>
+          {simulationsList?.map(({ id, creationDate, outputs }, index) => (
+            <div
+              role="button"
+              tabIndex={0}
+              key={index}
+              className={cx(SIMULATION_ITEM_CLASSNAME, {
+                retained: retainedSimulationIndex === index,
+                selected: selectedSimulationIndex === index,
+                anyRetained: retainedSimulationIndex !== -1,
+              })}
+              onClick={() => onSelectSimulation(index)}
+            >
+              <div className="simulation-name">
+                <div>
+                  {outputs
+                    ? t('simulationName.withOutputs', { id })
+                    : t('simulationName.withoutOutputs')}
+                </div>
+                {retainedSimulationIndex === index && (
+                  <span className="check-circle">
+                    <CheckCircle variant="fill" />
+                  </span>
+                )}
+              </div>
+              <div className="creation-date">
+                <span key={index}>
+                  {t('formatCreationDate', formatDateToString(creationDate, true))}
+                </span>
+              </div>
+              {selectedSimulationIndex === index && (
+                <div className="selected-simulation-indicator" />
+              )}
+            </div>
+          ))}
+        </div>
+        {showRightBtn && (
+          <div
+            className="scroll-btn right"
+            role="button"
+            tabIndex={0}
+            aria-label="Scroll right"
+            onClick={scrollRight}
+          >
+            <span>
+              <ChevronRight />
```

Comment:
Comparing to the sketch, I think we need to use the large icon (same for left chevron)

```suggestion
              <ChevronRight size='lg' />
```

Reply:
Done